js/main-menu.js

```diff
@@ -0,0 +1,129 @@
+(function ($, _) {
+  $(document).ready(function () {
+    amendMarkupOfMenuItemsWithFontAwesomeIcons();
+    customizeQuickSearchField();
+  });
+
+  /**
+   * Amends the markup of any menu item with a FontAwesome icon
+   */
+  function amendMarkupOfMenuItemsWithFontAwesomeIcons() {
+    menuItemsWithFontAwesomeIcon().each(function () {
+      var $menuItem = jQuery(this);
+
+      $('.fa', $menuItem).addClass('menumain-icon');
+      wrapMenuItemLabelInHTML($menuItem);
+    });
+  }
+
+  /**
+   * Changes the placeholder text of the quicksearch field
+   */
+  function changeQuickSearchFieldPlaceholder () {
+    $('#crm-qsearch .ui-autocomplete-input').attr('placeholder', 'Quick Search');
+  }
+
+  /**
+   * Customizes the quick search field
+   */
+  function customizeQuickSearchField () {
+    changeQuickSearchFieldPlaceholder();
+    manageCustomClassOfQuickSearchField();
+  }
+
+  /**
+   * Checks if the user has clicked outside of the quick search field
+   * by analyzing the given click target
+   *
+   * @param {Element} target
+   * @return {boolean}
+   */
+  function hasUserClickedOutsideQuickSearchField (target) {
+    var $target = $(target);
```

Comment:
I wonder why sometimes `$` used and sometimes `jQuery`?

Reply:
mistake, fixing it